src/main/java/pl/edu/wszib/controller/CarController.java

```diff
@@ -1,16 +1,60 @@
 package pl.edu.wszib.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import pl.edu.wszib.domain.entity.CarType;
 
 @RestController
-@RequestMapping("/car")
+@RequestMapping("/cars")
 public class CarController {
 
-    @RequestMapping("/hello")
+    //TODO: write implementation to all those methods
+
+    @RequestMapping(value = "/{id}")
+    @GetMapping
+    public String getCarById(@PathVariable("id") Integer id) {
+        return "car by id";
+    }
+
+    @RequestMapping(value = "/{brand}")
+    @GetMapping
+    public String getCarByBrand(@PathVariable("brand") String brand) {
+        return "car by brand";
+    }
+
+    @RequestMapping(value = "/brand/{model}")
+    @GetMapping
```

Comment:
tutaj nie wiem czy chcesz pobierać po brand i model, czy tylko po model. Jak po obu to brand też musisz zadeklarować jako argument, a jeśli tylko po modelu to brand można wyrzucić z url.